vra7/resource_vra7_deployment.go

```diff
@@ -347,27 +356,19 @@ func resourceVra7DeploymentRead(d *schema.ResourceData, meta interface{}) error
 	}
 
 	resourceDataMap := make(map[string]map[string]interface{})
+	var resourceViewsData []map[string]interface{}
 	for _, resource := range requestResourceView.Content {
-		if resource.ResourceType == sdk.InfrastructureVirtual {
```

Comment:
If you would like to not change these APIs, rather making them additive. I can add that back in using the generics implemented as part of this PR.

Reply:
Right now, this would be a breaking change. I can implement the above to ensure that it is not a breaking change -- I just don't have anyway to test anything defined in the examples directory (seems it uses a vRA instance I don't have access to).

Reply:
based on #21 this is not actually a breaking change and is completely additive!